### PR TITLE
Move integrations to registry

### PR DESCRIPTION
**Pull request by svrnm**
Major rework for the registry, this moves all integrations into the registry, by that data gets de-duplicated, and reusable. Ultimately we should do the same with vendors & adopters, to make the registry our "single source of truth", but step by step.

I also restructured the integrations page a little bit, splitting libraries from "application integrations"

Preview Integrations: https://deploy-preview-4991--opentelemetry.netlify.app/ecosystem/integrations/
Preview Registry: https://deploy-preview-4991--opentelemetry.netlify.app/ecosystem/registry/

FYI, this tags all maintainers because I had to restructure the standardized "libraries" page

### CODE REVIEW

File: content/en/docs/languages/erlang/libraries.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ weight: 40
 cSpell:ignore: Ecto Hex
 ---
 
-{{% docs/languages/libraries-intro "Erlang" %}}
+{{% docs/languages/libraries-intro "erlang" %}}
```
Comment (chalin) [inline content/en/docs/languages/erlang/libraries.md:8]: Nit, which you don't need to fix, but thought that I'd mention:

```suggestion
{{% docs/languages/libraries-intro erlang %}}
```

**Review by chalin — APPROVED** [review]
Wow, quite the refactoring, and improvements! RSLGTM 🎉

**Review by XSAM — APPROVED** [review]
It looks great!

File: data/registry/application-integration-python-cisco-nso.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  - nso
+  - python
+license: Commercial
+description:
+  The NSO Observability Exporter supports export of progress traces using OTLP.
+authors:
+  - name: Cisco
+urls:
+  website: https://www.cisco.com/c/en/us/products/cloud-systems-management/network-services-orchestrator/index.html
+  docs: https://developer.cisco.com/docs/nso/#!observability-exporter/
```
[the remark this answers is not in the document]
Comment (svrnm) [reply data/registry/application-integration-python-cisco-nso.yml:18]: yeah, let's fix that!

**Pull request merged**